src/Process/TiledNodeProcessing.js

```diff
@@ -0,0 +1,117 @@
+import BoundingBox from 'Scene/BoundingBox';
+
+import * as THREE from 'three';
+
+function subdivisionBoundingBoxes(bbox) {
+    const northWest = new BoundingBox(bbox.west(), bbox.center.x, bbox.center.y, bbox.north(), 0, 0, bbox.minCoordinate.unit);
+    const northEast = new BoundingBox(bbox.center.x, bbox.east(), bbox.center.y, bbox.north(), 0, 0, bbox.minCoordinate.unit);
+    const southWest = new BoundingBox(bbox.west(), bbox.center.x, bbox.south(), bbox.center.y, 0, 0, bbox.minCoordinate.unit);
+    const southEast = new BoundingBox(bbox.center.x, bbox.east(), bbox.south(), bbox.center.y, 0, 0, bbox.minCoordinate.unit);
+
+
+    return [northWest, northEast, southWest, southEast];
+}
+
+function requestNewTile(scheduler, geometryLayer, bbox, parent, level) {
+    const command = {
+        /* mandatory */
+        requester: parent,
+        layer: geometryLayer,
+        priority: 10000,
+        /* specific params */
+        bbox,
+        type: geometryLayer.nodeType,
+        level,
+        redraw: false,
+    };
+
+    return scheduler.execute(command);
+}
+
+function subdivideNode(context, layer, node) {
+    if (!node.pendingSubdivision2 && node.children.filter(n => n.layer == layer.id).length == 0) {
+        const bboxes = subdivisionBoundingBoxes(node.bbox);
+        node.pendingSubdivision2 = true;
+
+        const promises = [];
+        for (let i = 0; i < bboxes.length; i++) {
+            promises.push(
+                requestNewTile(context.scheduler, layer, bboxes[i], node).then(child => layer.initNewNode(context, node, child)));
+        }
+
+        Promise.all(promises).then(() => {
+            node.pendingSubdivision2 = false;
+            context.scene.notifyChange(0, true);
+        });
+    }
+}
+
+const _promises = [];
+export function initTiledGeometryLayer(context, layer) {
+    if (_promises.length > 0) {
+        return;
+    }
+
+    layer.root = [];
+
+    for (let i = 0; i < layer.schemeTile.rootCount(); i++) {
+        _promises.push(
+            requestNewTile(context.scheduler, layer, layer.schemeTile.getRoot(i), undefined, 0));
+    }
+    Promise.all(_promises).then((roots) => {
+        layer.root = roots;
+        for (const root of roots) {
+            // TODO: support a layer.root attribute, to be able
+            // to add a layer to a three.js node
+            context.scene.gfxEngine.scene3D.add(root);
```

Comment:
Sort of.
For this layer it looks similar. Other layers (eg wfs) doesn't have any layer.root attribute (nor need them).